src/main/java/database/service/entity/Course.java

```diff
@@ -0,0 +1,44 @@
+package database.service.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Course extends BaseEntity {
+    private String name;
+    private final List<Student> students = new ArrayList<>();
```

Comment:
Why this is final ? I think students list can be changed if new student is added or excluded from the class.